daprdocs/content/en/operations/monitoring/tracing/open-telemetry-collector.md

```diff
@@ -66,11 +63,7 @@ That's it! There's no need include any SDKs or instrument your application code.
 
 > **NOTE**: You can register multiple tracing exporters at the same time, and the tracing logs are forwarded to all registered exporters.
 
-Deploy and run some applications. After a few minutes, you should see tracing logs appearing in your Application Insights resource. You can also use **Application Map** to examine the topology of your services, as shown below:
-
-![Application map](/images/open-telemetry-app-insights.png)
-
-> **NOTE**: Only operations going through Dapr API exposed by Dapr sidecar (e.g. service invocation or event publishing) are displayed in Application Map topology.
+Deploy and run some applications. Wait for the trace to propagate to your tracing backend and view them there.
 
 ## Tracing configuration
```

Comment:
Same comment here. To remove this repeated section and add a link to the configuration article

Reply:
Done